templates/email/default/inactive-account.txt

```diff
@@ -0,0 +1,16 @@
+Subject: Your inactive account on [% site_name %]
+
+Hello [% user.name %],
+
+Your account on [% site_name %] has been inactive for [% email_from %]
+[% nget('month', 'months', email_from) %], and we automatically remove
+accounts that have been inactive after [% anonymize_from %]
+[% nget('month', 'months', anonymize_from) %]. If you wish to keep your
+account, please log in to the site and that will keep it active.
```

Comment:
It'd be handy to have a login link here. Also, no HTML version?

Reply:
Will do HTML version. A login link as in just a link to `/my` or one that logs you in? I think might not want the latter (risk of clicking by mistake, but former sure).